keanu-project/src/main/java/io/improbable/keanu/network/BayesianNetwork.java

```diff
@@ -1,9 +1,6 @@
 package io.improbable.keanu.network;
 
-import java.util.Collection;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Are we ok with wild card imports?

Reply:
See https://improbableio.atlassian.net/wiki/spaces/~616259109/pages/283574476/IntelliJ+settings for disabling wild card imports